README.md

```diff
@@ -1126,6 +1130,30 @@ err = accessManager.AssignRepoToProject("repoName", "tstprj", true)
 err = accessManager.AssignRepoToProject("repoName")
 ```
 
+#### Get all groups assigned to a project
+```go
+err = accessManager.GetProjectsGroups("tstprj")
```

Comment:
The documented API returned values should be corrected.

Let's rename "tstprj" to "my-project".